test/services/ProfessionalBodiesServiceSpec.scala

```diff
@@ -73,6 +73,30 @@ class ProfessionalBodiesServiceSpec extends SpecBase with MockitoSugar with Scal
             result.map(_ mustBe a[ProfessionalBody])
         }
       }
+
+      "provided bad data return an exception as errors occur" in {
+        val result = professionalBodiesService.localSubscriptions("test-professional-bodies.json")
+
+        val exception = intercept[Exception] {
+          whenReady(result) {
+            _ mustBe an[Exception]
+          }
+        }
+
```

Comment:
After having a look at this it appeared that the '_ mustBe an[Exception]' part of this is not working as intended and will always pass no matter the value. Would recommend changing this to the following:

          whenReady(result.failed) {
            result =>
              result mustBe an[Exception]
              result.getMessage must include("failed to parse bodies")
          }